src/HonzaBotner.Discord.Services/Commands/VoiceCommands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using HonzaBotner.Discord.Services.Attributes;
+
+namespace HonzaBotner.Discord.Services.Commands
+{
+    [Group("voice")]
+    [Description("Commands to control custom voice channels.")]
+    [RequireChannel(750108543125946448)]
+    public class VoiceCommands : BaseCommandModule
+    {
```

Comment:
```suggestion
    {
        internal const ulong RequiredChannel = 750108543125946448;
        
```

Reply:
Maybe it would be better to make logic in the method's body, so we can use config... I don't like random constants.